examples/webpack.config.react.js

```diff
@@ -1,7 +1,7 @@
 const webpack = require('webpack')
 const HtmlWebpackPlugin = require('html-webpack-plugin')
 const path = require('path')
-const dotenv = require('dotenv').config()
```

Comment:
With this change, are we actually loading the `.env` file anywhere?

Something that could be considered https://www.npmjs.com/package/dotenv-safe

Reply:
Interesting.  I assumed lines 59-62 are pulling from .env, and that assumption is consistent with the Issue.  Any reason to consider changing this over to dotenv instead of how it's done here already?

Reply:
Ah, didn't mean to remove that line.  This line loads .env into process.env, then the custom plugin verifies that .env exists...
Good catch

Reply:
So `dotenv` is what is actually loading from the `.env` file. It also allows you to override what is set in the file by setting an environment variable before running the process. So adding it back is the right move.

As for switching to `dotenv-safe` vs what you have here, I have no real preference either way.

Reply:
You beat me to it, Github needs a notification that someone has commented while you're preparing a comment so you know to refresh first.